.github/workflows/build.yml

```diff
@@ -1,4 +1,4 @@
-name: psabpf CI
+name: static build psabpf
```

Comment:
Why do we call it "static build psabpf" while we actually build both static and shared in this workflow? I suggest naming it "Build psabpf".